pkg/rancher-desktop/utils/__tests__/paths.spec.ts

```diff
@@ -134,7 +134,7 @@ describe('paths', () => {
     const platform = os.platform();
 
     if (['darwin', 'linux'].includes(platform)) {
-      expect(pathsToDelete.some( dir => paths.lima.startsWith(dir))).toEqual(platform === 'darwin');
+      expect(pathsToDelete.some( dir => paths.lima.startsWith(dir))).toBeTruthy();
```

Comment:
Not sure about the point of this test. Isn't is effectively verifying that

```
paths.lima === filepath.Join(paths.appHome, "lima")
```

Not sure why we need to check all the other directories. Not even sure if we need this test at all.

Reply:
Yeah, we no longer need this test now that `paths.appHome` means the same thing on linux and macOS.